k8s/README.md

```diff
@@ -54,6 +54,8 @@ To use the development image for testing with specific version of VPP, see
 
   * Node configuration (section `NodeConfig`; one entry for each node)
     - `NodeName`: name of a Kubernetes node;
+    - `UseDhcpOnMainInt`: acquire IP address for the main VPP interface using DHCP 
+        (beware: renew of IP address is not supported)
```

Comment:
is renew lease or changing ip ?

Reply:
The IP address could be changed when the lease is renewed. I guess that's what Lukas is referring to. If the lease is renewed, but the IP address stays the same, we should be fine.

Reply:
yes correct, the change of IP address is not supported. I'll re-phrase it